.travis.yml

```diff
@@ -3,8 +3,10 @@ sudo: false
 language: go
 
 go:
-- 1.5.4
-- 1.6.2
+- 1.8.x
+- tip
```

Comment:
`tip` is kinda deprecated, it's better to call it `master`: https://github.com/travis-ci/gimme/pull/102